inc/class-registration.php

```diff
@@ -84,6 +84,7 @@ public function init() {
 		add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_block_editor_assets' ) ); // Don't change the priority or else Blocks CSS will stop working.
 		add_action( 'enqueue_block_assets', array( $this, 'enqueue_block_assets' ) );
 		add_filter( 'render_block', array( $this, 'load_sticky' ), 900, 2 );
+		add_filter( 'render_block', [ $this, 'subscribe_fa' ], 900, 2 );
```

Comment:
By having this hook here, the `subscribe_fa` function is triggered. But the action `add_action(
			'get_footer',` is not getting the updated value.

A better solution is the integration in the pipeline as Templates.